app/api/auth.ts

```diff
@@ -27,6 +27,13 @@ function parseApiKey(bearToken: string) {
 
 export function auth(req: NextRequest) {
   const authToken = req.headers.get("Authorization") ?? "";
+  const aoaiApiKey = req.headers.get("azure-api-key") ?? "";
```

Comment:
Just use `azureApiKey` will be clearer.

Reply:
Thanks your opinion. Given the compatibility of the different versions of HTTP protocols and services, I think a case-insensitive spinal-case naming method is a better choice in http header. This is also the recommended name by many server-side frameworks.